packages/select/test/select.test.js

```diff
@@ -184,14 +185,14 @@ describe('vaadin-select', () => {
       it('should select items when alphanumeric keys are pressed', () => {
         select.opened = false;
         expect(select._menuElement.selected).to.be.equal(2);
-        keyDownChar(input, 'o');
-        keyDownChar(input, 'p');
-        keyDownChar(input, 't');
+        keyDownChar(button, 'o');
```

Comment:
How about `keyDownChar(this._valueNode)`? Would it make more sense here?

Reply:
...or `keyDownChart(valueButton)`?

Reply:
Done.